tests/Api/Shop/CheckoutCompletionTest.php

```diff
@@ -39,7 +38,7 @@ public function it_prevents_from_order_completion_if_order_is_in_the_cart_state(
         $this->client->request(
             method: 'PATCH',
             uri: sprintf('/api/v2/shop/orders/%s/complete', $tokenValue),
-            server: ContentType::APPLICATION_JSON_MERGE_PATCH,
+            server: $this->headerBuilder()->withMergePatchJsonContentType()->withJsonLdAccept()->build(),
```

Comment:
Would be cool to use same strategy, either build heders per method call as it's done for this file or use `private buildHeaders` as in `tests/Api/Admin/OrdersTest.php`